src/containers/App/App.js

```diff
@@ -88,29 +88,11 @@ import {
   isReadOnly,
   isWebSocketConnected
 } from '../../reducers';
-import messages from '../../nls/messages_en.json';
 
 import '../../scss/App.scss';
 
-/* istanbul ignore next */
-if (process.env.I18N_PSEUDO) {
-  const startBoundary = '[[%';
-  const endBoundary = '%]]';
-  // Make it easier to identify untranslated strings in the UI
-  Object.keys(messages).forEach(lang => {
-    const messagesToDisplay = messages[lang];
-    Object.keys(messagesToDisplay).forEach(messageId => {
-      if (messagesToDisplay[messageId].startsWith(startBoundary)) {
-        // avoid repeating the boundaries when
-        // hot reloading in dev mode
-        return;
-      }
-      messagesToDisplay[
-        messageId
-      ] = `${startBoundary}${messagesToDisplay[messageId]}${endBoundary}`;
-    });
-  });
-}
+import { locales } from '../../../config_frontend/config.json'; 
+const { default: defaultLocale, supported: supportedLocales } = locales;
```

Comment:
The test failures are caused by 2 linting errors:
- trailing whitespace on the import line
- missing blank line after last import

```suggestion
import { locales } from '../../../config_frontend/config.json';

const { default: defaultLocale, supported: supportedLocales } = locales;
```